src/test/java/org/prebid/server/bidder/yahoossp/YahooSSPBidderTest.java

```diff
@@ -44,14 +56,25 @@ public class YahooSSPBidderTest extends VertxTest {
 
     private YahooSSPBidder yahooSSPBidder;
 
+    @Rule
+    public final MockitoRule mockitoRule = MockitoJUnit.rule();
+
+    @Mock
+    private BidRequestOrtbVersionConversionManager conversionManager;
+
     @Before
     public void setUp() {
-        yahooSSPBidder = new YahooSSPBidder(ENDPOINT_URL, jacksonMapper);
+        BidRequestOrtb26To25Converter innerConverter = new BidRequestOrtb26To25Converter(jacksonMapper);
```

Comment:
I removed the usage of the converter. Since we don't check the actual conversion and only our custom regs changes we mock the conversion by always returning the input each test builds.